tests/buffer/associative_containers.cpp

```diff
@@ -10,7 +10,7 @@
 
 using namespace cl::sycl;
 
-int test_main(int argc, char *argv[]) {
+int test_main(int /*argc*/, char** /**argv[]*/) {
```

Comment:
No, nothing strong. I just never wrote plain char*[] as a type before and the linker was arguing about char** missing as a signature type of test_main, so I just went along with what the linker suggested. What's the harm if it's unused anyway?

Anyhow, changed all occurences with `char*[] /*argv*/`.